src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.DukeListOobException;
+import duke.exceptions.DukeWrongInputException;
+
+public class TaskList {
+
+    private ArrayList<Task> tasks;
```

Comment:
Considering that the methods will be invoked on a `TaskList`, I think that prefixing `list` to each of the methods in this class is redundant, and potentially misleading in the case of `listDelete` which suggests that it deletes an entire list.